doc/ale.txt

```diff
@@ -1061,6 +1062,24 @@ g:ale_yaml_yamllint_options                       *g:ale_yaml_yamllint_options*
 
   This variable can be set to pass additional options to yamllint.
 
+------------------------------------------------------------------------------
+4.32. cmakelint                                  *ale-linter-options-cmakelint*
```

Comment:
This is okay. Sorting the sections is a pain, because you need to change all of the section numbers, as you say.

I might sort the sections again some day.